x/wasm/keeper/querier_test.go

```diff
@@ -104,7 +114,7 @@ func TestQueryAllContractState(t *testing.T) {
 		t.Run(msg, func(t *testing.T) {
 			got, err := q.AllContractState(sdk.WrapSDKContext(ctx), spec.srcQuery)
 			if spec.expErr != nil {
-				require.True(t, errors.Is(err, spec.expErr), "but got %+v", err)
+				require.Equal(t, spec.expErr, err, "but got %+v", err)
```

Comment:
use https://pkg.go.dev/github.com/stretchr/testify/require#EqualError if able

Reply:
There are other errors in this test function like status.Errors, types.ErrNotFound. So I think it's not enough if just use EqualError to compare error strings.